realpath_turbo.c

```diff
@@ -118,6 +113,12 @@ PHP_RINIT_FUNCTION(realpath_turbo)
 
 PHP_MINIT_FUNCTION(realpath_turbo)
 {
+#if PHP_MAJOR_VERSION < 8
+        zend_disable_function("link", sizeof("link")-1 TSRMLS_CC);
```

Comment:
I am wondering about this change. Looks like we would now always disable these functions...

Reply:
I have add a test for this.

Reply:
In short, function from configuration are disabled at PHP startup, additional ones by the plugin during MINIT.

The ini stuff is only useful if some user check configuration, for consistency

Reply:
Sorry, I don't read enough your question...

> Looks like we would now always disable these functions...

Indeed... grr

Reply:
Should be ok now, + new test